day2_finalproject_tipcalculator.py

```diff
@@ -1,7 +1,16 @@
+print("******************************")
 print("Welcome to the tip calculator.")
 
 total_bill = input("What was the total bill? ")
 tip_amount = input("How much would you like to tip? 10, 12, 15 or 18? ")
+
+
+#some bounds on tipping 
+if(int(tip_amount)<0):
+    print("Come on you have to tip something")
+elif(int(tip_amount)>100):

```

Comment:
What if:
elif(int(tip_amount)>total_bill)

Reply:
I can do this!